server/src/main/java/org/eclipse/lsp/cobol/core/visitor/VariableDefinitionDelegate.java

```diff
@@ -297,7 +306,7 @@ private int retrieveOccursTimes(@NonNull DataOccursClauseContext dataOccursClaus
   @NonNull
   private List<IndexItem> retrieveIndexItem(int levelNumber, @NonNull DataOccursClauseContext clause) {
     return clause.indexName().stream()
-        .map(it -> new IndexItem(levelNumber, it.getText().toUpperCase(), positions.get(it.start)))
+        .map(it -> new IndexItem(levelNumber, it.getText().toUpperCase(), positions.get(it.start), false))
```

Comment:
Yes, you are right. Here is the initial creation of the IndexItem. Later on, it will be recreated via the `updatePatern` method and it sets the right global value.